org.scala-ide.sdt.core/src/org/scalaide/core/compiler/IScalaPresentationCompiler.scala

```diff
@@ -367,6 +367,10 @@ object IScalaPresentationCompiler extends HasLogger {
                 Thread.currentThread().interrupt()
                 None
 
+              case Right(f: FreshRunReq)          =>
+                logger.info("FreshRunReq in ask:\n" + f)
```

Comment:
`info` has an overload, which takes in its second argument a throwable. Please choose that one.